README.md

```diff
@@ -73,6 +73,18 @@ gem install specific_install
 gem specific_install -l https://github.com/CDLUC3/uc3-ssm
 ```
 
+#### Rails installation:
+To install the gem in a Rails application:
+1. add the following to your Gemfile ```ruby
```

Comment:
I had to embed a github credential into my bundler config file to use this reference to the gem.  I did not need to do that for a git based reference.  Did you encounter a similar dependency?